backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/jni/RustBackend.kt

```diff
@@ -317,6 +317,22 @@ class RustBackend private constructor(
         )
     }
 
+    override suspend fun proposeTransferFromUri(
+        account: Int,
+        uri: String
+    ): ProposalUnsafe =
+        withContext(SdkDispatchers.DATABASE_IO) {
+            ProposalUnsafe.parse(
+                proposeTransferFromUri(
+                    dataDbFile.absolutePath,
+                    account,
+                    uri,
+                    networkId = networkId,
+                    useZip317Fees = IS_USE_ZIP_317_FEES
```

Comment:
This shouldn't be a parameter any longer; transactions that do not use ZIP 317 fees should be expected to never be mined at this point.